hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -314,165 +314,168 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
       keys.push('Entity Type');
       values.push(summaryResponse.type);
 
-      if (summaryResponse.countStats.type === 'KEY') {
-        const keyEndpoint = `/api/v1/namespace/du?path=${path}&replica=true`;
-        const { request: metadataRequest, controller: metadataNewController } = AxiosGetHelper(keyEndpoint, cancelKeyMetadataSignal);
-        cancelKeyMetadataSignal = metadataNewController;
-        metadataRequest.then(response => {
-          keys.push('File Size');
-          values.push(byteToSize(response.data.size, 3));
-          keys.push('File Size With Replication');
-          values.push(byteToSize(response.data.sizeWithReplica, 3));
-          console.log(values);
-
-          this.setState({
-            showPanel: true,
-            panelKeys: keys,
-            panelValues: values
+      // If status is not INITIALIZING then adding Entities for metadata to avoid null failures on UI.
```

Comment:
Extending on @ArafatKhan2198's comment, could you change the comment to be a bit more explaining.
```suggestion
// If status is INITIALIZING, we cannot add entities for metadata as it will cause null failures
// Hence we only add Entities if the status is not INITIALIZING
```

Reply:
@devabhishekpal 
Done the changes in latest commit 23ede0494cd9566ccc676fa3348b3343a6b52462.